aspnetcore/data/ef-rp/read-related-data.md

```diff
@@ -31,57 +32,87 @@ The following illustrations show the completed pages for this tutorial:
 
 There are several ways that EF can load related data into the navigation properties of an entity:
 
-* Eager loading. When the entity is read, its related data is retrieved. This typically results in a single join query that retrieves all of the data that's needed. Eager loading is specified in EF with the `Include` and `ThenInclude` methods.
+* [Eager loading](https://docs.microsoft.com/ef/core/querying/related-data#eager-loading). Eager loading is when a query for one type of entity also loads related entities. When the entity is read, its related data is retrieved. This typically results in a single join query that retrieves all of the data that's needed. Eager loading is specified with the `Include` and `ThenInclude` methods.
 
-  ![Eager loading example](read-related-data/_static/eager-loading.png)
+ ![Eager loading example](read-related-data/_static/eager-loading.png)
 
-  zz
-  
-Rather than a single join query, some of the data can be retrieved in separate queries, and EF "fixes up" the navigation properties. "fixes up" means that EF automatically adds the related entities where they belong. Eager loading with separate queries results in multiple queries sent to the DB.
+ The data can be retrieved in separate queries, and EF "fixes up" the navigation properties. "fixes up" means that EF automatically populates the navigation properties. Eager loading with separate queries results in multiple queries sent to the DB.
```

Comment:
When using separate queries with "Load" this is more like explicit loading than eager loading.

Reply:
Although I agree it is somewhat ambiguous. Maybe it should just be its own section?

Reply:
@ajcvickers  I made a section for this.